src/zenml/stack/flavor_registry.py

```diff
@@ -115,8 +115,8 @@ def _register_flavor(
             id=UUID(int=0),
             user=client.active_user,
             project=client.active_project,
-            created=datetime.now(),
-            updated=datetime.now(),
+            created=datetime.utcnow(),
```

Comment:
We use `datetime.now` in some new model creation/update calls since the tfx merge, I'm guessing those should be updated as well to be consistent?

Reply:
I'll give that a new search and fix them if they need fixing.

Reply:
done